.eslintrc

```diff
@@ -5,9 +5,8 @@
     "browser": true,
     "node": true
   },
-
   "rules": {
-    "react/no-find-dom-node": 1,
-    "react/no-string-refs": 1
+    "react/no-unused-prop-types": 1,
+    "jsx-a11y/no-static-element-interactions": 1
```

Comment:
So `no-unused-prop-types` is broken which is why that's a warn right now (nested prop types don't check properly). Um, rn I don't know if I should go in and change the interactive elements in the datepicker to be buttons as they should be given that I don't feel like that would be a good keyboard navigable experience yet.